vendor/github.com/containers/image/v5/copy/copy.go

```diff
@@ -1039,11 +1041,10 @@ type diffIDResult struct {
 
 // copyLayer copies a layer with srcInfo (with known Digest and Annotations and possibly known Size) in src to dest, perhaps compressing it if canCompress,
 // and returns a complete blobInfo of the copied layer, and a value for LayerDiffIDs if diffIDIsNeeded
-func (ic *imageCopier) copyLayer(ctx context.Context, srcInfo types.BlobInfo, toEncrypt bool, pool *mpb.Progress) (types.BlobInfo, digest.Digest, error) {
+func (ic *imageCopier) copyLayer(ctx context.Context, srcInfo types.BlobInfo, toEncrypt bool, pool *mpb.Progress, options *Options) (types.BlobInfo, digest.Digest, error) {
```

Comment:
If `pullAlways` needs propagating all the way down here, please make it a member of `copier` instead of an extra parameter that most functions in the stack don’t need.